crates/ruff_linter/src/rules/pep8_naming/rules/camelcase_imported_as_acronym.rs

```diff
@@ -97,3 +90,34 @@ pub(crate) fn camelcase_imported_as_acronym(
     }
     None
 }
+
+fn is_ignored_because_of_import_convention(
+    asname: &str,
+    stmt: &Stmt,
+    alias: &Alias,
+    checker: &Checker,
+) -> bool {
+    let full_name = if let Some(import_from) = stmt.as_import_from_stmt() {
```

Comment:
```suggestion
    let full_name = if let Stmt::ImportFrom(import_from) = stmt {
```

Reply:
I prefer the `as_import_from_stmt`

Reply:
Interesting -- why? To me it seems like it's more verbose and adds unnecessary indirection here. Obviously not a big issue, just curious!